usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/WashData.java

```diff
@@ -0,0 +1,41 @@
+package org.pesmypetcare.usermanagerlib.datacontainers;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+/**
+ * @author Marc Simó
+ */
+public class WashData {
```

Comment:
All the methods are missing their javadoc

Reply:
This class will be changed so I will comment it once it's finished